### PR TITLE
Refactor:Order Service Testing

### PR DESCRIPTION
**Pull request by kai890707**
**Description**

1. Refactor order service testing.

**Checklist:**
- [ ] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] Conforms to style guide

### CODE REVIEW

File: develop/tests/API/v2/OrderTest.php [hunk 1]
```diff
@@ -143,14 +162,53 @@ public function testIndex()
...
+    }
+
+    /**
+     * @test
+     *
+     * [SUCCESS CASE] Get all order without parameters
+     *
+     * @return void
+     */
+    public function testIndexOrderNotHasParametersSuccess()
```
Comment (ping-yee) [inline develop/tests/API/v2/OrderTest.php:203]: Change Not has to without.

File: develop/tests/API/v2/OrderTest.php [hunk 2]
```diff
@@ -171,8 +231,40 @@ public function testIndex()
...
+    }
+
+    /**
+     * @test
+     *
+     * [FAIL CASE] Get all order without parameters
+     *
+     * @return void
+     */
+    public function testIndexOrderNotHasParametersFail()
```
Comment (ping-yee) [inline develop/tests/API/v2/OrderTest.php:243]: Ditto.

**Review by ping-yee — CHANGES_REQUESTED** [review]
There are some little changes need to be handled.

File: develop/tests/API/v2/OrderTest.php (hunk 1 above)
Comment (kai890707) [reply develop/tests/API/v2/OrderTest.php]: This change is done.

File: develop/tests/API/v2/OrderTest.php (hunk 2 above)
Comment (kai890707) [reply develop/tests/API/v2/OrderTest.php]: This change is done.

**Review by ping-yee — COMMENTED** [review]
LGTM

**Review by ping-yee — APPROVED** [review]
LGTM

**Pull request merged**